step.yml

```diff
@@ -41,67 +41,66 @@ toolkit:
     package_name: github.com/bitrise-steplib/bitrise-step-android-build-for-ui-testing
 
 inputs:
-  - project_location: $BITRISE_SOURCE_DIR
-    opts:
-      title: Project Location
-      summary: "The root directory of your android project, for example, where your root build gradle file exist (also gradlew, settings.gradle, etc...)"
```

Comment:
You have removed the summary section for the `project_location` but kept it for the `module` and `variant`. The text is exactly the same as in the `description` field. Should we remove those all of the `summaries`?

(Probably a different discussion but as I can see those two fields almost always hold the same value. Does it bring any value? We could just have the `summary` by default and if we want to go into more details then use the `description` field.)

Reply:
Thanks for catching this! You are right, our convention is to only have a description field if it has more details than the summary. I should have deleted the `description` field and not `summary`. I'm fixing this soon.

Reply:
@tothszabi please take a look now!